src/interp.cc

```diff
@@ -2226,6 +2245,12 @@ Result Thread::Run(int num_instructions) {
         break;
       }
 
+      case Opcode::I8X16Splat: {
+        uint8_t dLane = static_cast<uint8_t>(Pop<uint32_t>());
```

Comment:
nit: style for local variables is `snake_case`, no hungarian notation. Same for above

Reply:
Got it.

Reply:
Done.